Wabbajack/AppState.cs

```diff
@@ -175,6 +242,30 @@ private void VisitNexusSite()
             }
         }
 
+        private ModlistPropertiesWindow modlistPropertiesWindow;
+        internal string newImagePath;
```

Comment:
I've noticed `internal` being used around, which is N/A in this project's context
Essentially `internal` means "`public` for this library, `private` for users of this library"

But since we're writing a GUI, there's never going to be any users of this project as a library, so any `internal` are basically just `public`.  Not a gamebreaking issue, but just thought I'd mention it